File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToCancelRequestConverter.java
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request;

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.CancelRequest;
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel;
import org.springframework.core.convert.converter.Converter;
import org.springframework.stereotype.Component;

@Component
public class EntryStateToCancelRequestConverter implements Converter<CustomEntryStateModel, CancelRequest> {

    @Override
    public CancelRequest convert(CustomEntryStateModel entryStateModel) {
        return new CancelRequest();
    }
}

File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToCancelRequestConverter.kt
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.CancelRequest
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel
import org.springframework.core.convert.converter.Converter
import org.springframework.stereotype.Component

@Component
class EntryStateToCancelRequestConverter : Converter<CustomEntryStateModel, CancelRequest> {

    override fun convert(entryStateModel: CustomEntryStateModel): CancelRequest {
        return CancelRequest()
    }
}
